tracer/src/Datadog.Trace/ISpanContextInjector.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="ISpanContextInjector.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+#nullable enable
+
+namespace Datadog.Trace
```

Comment:
why you no filescoped namespace 😢

Reply:
copy paste of the Extraction one I suppose